frontend/app/views/spree/products/_cart_form.html.erb

```diff
@@ -34,7 +34,7 @@
         <div id="product-price">
           <h6 class="product-section-title"><%= Spree.t(:price) %></h6>
           <div>
-            <span class="price selling" itemprop="price">
+            <span class="price selling" itemprop="price" content="<%= @product.price %>">
```

Comment:
`@product.price` will return something different than `display_price(@product)` in many circumstances (customer uses a different currency than the admins, for example). This would need to recreate the behavious of `display_price`.